fixtures/manufacturers.json

```diff
@@ -428,6 +428,9 @@
     "website": "https://www.soundlight.de/",
     "rdmId": 21324
   },
+  "spl": {
+    "name": "SPL"
```

Comment:
The name "SPL" doesn't appear on the product webpage at all.  It's listed as brand "TOM", manufacturer "‎Casisy", and I can't find any record of either of those existing, either.  They don't seem to have a webpage.